tmva/tmva/CMakeLists.txt

```diff
@@ -368,8 +364,26 @@ ROOT_STANDARD_LIBRARY_PACKAGE(TMVA
     Imt
     RIO
     XMLIO
-  INSTALL_OPTIONS
-    ${installoptions}
+)
+
+# Enable parts dependent RDataFrame
+if(dataframe)
+    list(APPEND TMVA_HEADERS
+        TMVA/RTensor.hxx
+        TMVA/RTensorUtils.hxx
+    )
+    list(APPEND TMVA_DEPENDENCIES
+        ROOTDataFrame
+        ROOTVecOps
+    )
+endif()
+
+ROOT_STANDARD_LIBRARY_PACKAGE(TMVA
+  HEADERS ${TMVA_HEADERS}
+  SOURCES ${TMVA_SOURCES}
+  DICTIONARY_OPTIONS -writeEmptyRootPCM ${dictoptions}
+  DEPENDENCIES ${TMVA_DEPENDENCIES}
+  INSTALL_OPTIONS ${installoptions}
 )
```

Comment:
Please set variables only for optional extras (look in `tree/dataframe/CMakeLists.txt` for an example). The above goes against the current coding style.

Reply:
Thanks, done!